.gitignore

```diff
@@ -3,4 +3,6 @@ felix.sh
 .classpath
 .project
 .settings/
-target/
\ No newline at end of file
+target/
+workspace/
+osgi-spring-felix/felix-cache
```

Comment:
The bundles folder contains the example bundle I use to programmatically install, so leaving that out of the ignore file was intentional.

The cache.lock must have got added before adding the ignore entry, I'll fix that up.